documentation.md

```diff
@@ -54,6 +54,10 @@ Totally hides the widget if set to true and no plan matches the purchase amount.
 
 Allow to choose which payment plan's tab will be displayed by default. It will have effect only if the selected plan is eligible. If an array is provided, it will select the first eligible plan from this array.
 
+- hideBorder: `boolean` [optional, default: true]
+
+Hide the border if set to false, set to true as default
```

Comment:
I think this line of the doc is also incorrect. 
It should be "hide when set to true" 👈 it's what the code does and what seams logic.

```suggestion
Hide the border if set to true, set to false as default
```👈